scripts/install_third_party.py

```diff
@@ -324,28 +323,10 @@ def download_manifest_files(filepath):
                     dependency_tar_root_name, dependency_target_root_name)
 
 
-MATHJAX_REV = '2.6.0'
```

Comment:
Why are we removing this? We still do not require `unpacked` and `fonts`, right?

Reply:
Hi @apb7,
`skip files` in `app.yaml `contains all the list of dirs which should be skipped deploying in the production i.e GAE. before having the method of skip_files we used to manually delete heavy sized dirs using `shutil.rm` hence, in the removed code deletes the `unpacked` and `fonts` folder of the MathJax. i've removed the entire code and added these dirs to the skip_files which does the same (skip_files don't delete but skips deploying into GAE ) this was suggested by @seanlip [here](https://github.com/oppia/oppia/pull/6778#discussion_r286650170)

Also, our file_size_check linter respects skip_files so the file count of unpacked and fonts are also removed 

Thanks!

Reply:
Oh, right! Missed this. Thanks for the explanation.